readme.md

```diff
@@ -985,6 +986,7 @@ Similar to `useApp`, it's equivalent to consuming `StdoutContext` directly.
 - [ink-divider](https://github.com/JureSotosek/ink-divider) - A divider component.
 - [ink-progress-bar](https://github.com/brigand/ink-progress-bar) - Configurable component for rendering progress bars.
 - [ink-table](https://github.com/maticzav/ink-table) - Table component.
+- [ink-ascii](https://github.com/hexrcs/ink-ascii) - Awesome text component with more font choices, based on Figlet.
```

Comment:
Have you tested your package on production?

You set `main: "index.js"` in your [package.json](https://github.com/hexrcs/ink-ascii/blob/master/package.json#L5) but your main file is `dist/index.js`.

Reply:
Sorry about that, I'm very new to library authoring. Thanks for pointing it out and fixing it!